HenHen.Framework/Graphics2d/RectangleF.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Numerics;
+
+namespace HenHen.Framework.Graphics2d
+{
+    public class RectangleF
+    {
+        public float Left { get; set; }
+        public float Right { get; set; }
+        public float Top { get; set; }
+        public float Bottom { get; set; }
+
+        public float Width => Right - Left;
```

Comment:
Dodać dokumentację, że właściwość będzie ujemna dla niektórych sytuacji.
Jak:
- wpisać `///`
- powinno pokazać się coś takiego:
```C#
/// <summary>
/// 
/// </summary>
```
- wypełnić w taki sposób
```C#
/// <summary>
/// If <see cref="Left"/> is greater than <see cref="Right"/>,
/// this will be negative.
/// </summary>
public float Width => Right - Left;
```